components/HeroNotificationBox.tsx

```diff
@@ -0,0 +1,21 @@
+import Arrow from "../public/arrow_forward.svg";
+import Image from "next/image";
+export default function HeroNotificationBox() {
```

Comment:
```suggestion
const HeroNotificationBox = () => {
```
Let's keep the app consistent and use arrow functions

Reply:
Fixed in recent commit